index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b21ce73c3c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div id="header_content">
+            <div id="logo">Progress Eze</div>
+            <nav>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    
+    <main id="main">
+        <h1>Hey There. I’m Progress</h1>
+        <h2>I’m a software developer</h3>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <div class="social_handels">
+            <div class="social_icons">
+                <a  class="icons"href="https://github.com/Progress2002"><i class="fa-brands fa-github"></i></a>
+                <a class="icons" href="https://www.linkedin.com/in/progress-ezeamaka-27b114247"><i class="fa-brands fa-linkedin-in"></i></a>
+                <a  class="icons" href="https://twitter.com/Progress_2002">    <i class="fa-brands fa-twitter"></i>
+                </a>
+                <a class="icons mail-icon" href="ezeamakachisom2002@gmail.com">M</a>
+            </div>
+        </div>
+        <div class="image img-1"><img src="images/Vector (1).png" alt="bg_img"></div>
+        <div class="image img-2"><img src="images/Vector (2).png" alt="bg_img"></div>
+        <div class="image img-3"><img src="images/Vector (3).png" alt="bg_img"></div>
+        <div class="image img-4"><img src="images/Vector (4).png" alt="bg_img"></div>
+        <div class="image img-6"><img src="images/Vector (6).png" alt="bg_img"></div>
+        <div class="image img-5"><img src="images/Vector (5).png" alt="bg_img"></div>
+        <div class="image img-7"><img src="images/Vector (7).png" alt="bg_img"></div>
+        <div class="image img-8"><img src="images/Vector(8).png" alt="bg_img"></div>
+        <div class="image img-9"><img src="images/Vector (9).png" alt="bg_img"></div>
+        <div class="image img-10"><img src="images/Vector (10).png" alt="bg_img"></div>
+        <div class="image img-11"><img src="images/Vector (11).png" alt="bg_img"></div>
+        <div class="image img-12"><img src="images/Vector (12).png" alt="bg_img"></div>
```

Comment:
- [ ] Kindly note that these images are purely for design purposes as they do not add any special meaning or interpretation to the webpage and as such should be added as a background image. Please check it out  👍🏽